### DISCUSSION TITLE
Exploring K_gl in MultiModel Hopf

### DISCUSSION THREAD

**Discussion by riccardoleone1991** [neurolib-dev/neurolib#227] [How to]
Hello there,

First of all, I would like to really thank you for developing Neurolib! It's a great resource for people like me starting with whole-brain modeling, I especially found the tutorials really useful! 
Then, to my specific issue. I am currently trying to use MultiModel and my current aim is to find the best coupling parameter for a model with fixed a and heterogeneous w for each node. I am not sure though how to evaluate different coupling parameters, as I don't find any K_gl in the specifics of the model (I only found "diffusive" in the specifics), but I would like to play around with different K_gl and find the best one for my subject set. Is there any way to do this? 
Thanks again!

Best,

Riccardo

**Comment by jajcayn** [comment]
Hi @riccardoleone1991 
Thanks for using `neurolib`:) `MultiModel` does not use `K_gl` as a global coupling parameter since it is just a scalar multiplier for the connectivity matrix. 
The best approach is to create a coupling parameter in the model definition artificially (as `model.params["coupling"] = 1.0`) and then use it in your evaluation function to multiply the connectivity matrix and then use this connectivity matrix in your model, that should work.

If you'd have any more questions, just let me know!
